GitHub-Test-4-PyNLPDiscuss/GitTestMDjms9354.md

```diff
@@ -0,0 +1,17 @@
+# Title
+
+    Question 1
+
+    What I find interesting about NLP is the large volume of textual data, I am curious as to how it measures a human idea and determines what is important.
```

Comment:
If NLP determines what is important within text, it might also be interesting to see what it doesn't consider important by comparison.